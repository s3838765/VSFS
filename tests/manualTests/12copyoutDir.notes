NOTES V1.0
=dir1/
@dir1/file1
 Some file data
@dir1/file2
