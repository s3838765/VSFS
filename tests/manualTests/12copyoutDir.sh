#! /usr/bin/sh
../../VSFS copyout 12copyoutDir.notes dir1/ 12copyoutDir.out 1> 12copyoutDir.out 2>&1
