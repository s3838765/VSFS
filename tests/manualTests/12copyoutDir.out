Copying directories is not supported. Terminating program.
